### ISSUE TITLE
better incoming call notifications

### ISSUE THREAD

**Issue by elad661** [holylobster/nuntius-linux#10]
Right now if I get  acall when nuntius is running, an "Incoming call" notification appears on my monitor, but it quickly disappears while the phone is still ringing. I guess this is due to Android removing the incoming call notification quickly after creating it when they show the call UI.

I think nuntius should add special-casing for incoming calls to make sure a notification (with an "urgent" hint) appears on my monitor and doesn't disappear until the phone stops ringing or I manaually dismiss it.

**Comment by nacho** [comment]
Yes, I completely agree about this, and we also had in mind integrating the call profile, so you could even anwer to a call directly from the computer.

**Comment by elad661** [comment]
It would be awesome if answering directly from my computer would use my computer microphone and speakers to make the call instead of the phone itself!
